### PR TITLE
Update Google Project memberships for Cluster API Core,  AWS and Azure projects

### PR DESCRIPTION
**Pull request by randomvariable**
To match current maintainers

### CODE REVIEW

**Review by justaugustus — CHANGES_REQUESTED** [review]
Two comments :)

File: groups/groups.yaml [hunk 1]
```diff
@@ -881,11 +881,12 @@ groups:
     settings:
       ReconcileMembers: "true"
     members:
+      - arudoi@newrelic.com
```
[the remark this answers is not in the document]
Comment (randomvariable) [reply groups/groups.yaml:884]: @rudoi , dm me your address and i'll update.

[the remark this answers is not in the document]
Comment (randomvariable) [reply groups/groups.yaml]: Updated.

File: groups/groups.yaml [hunk 2]
```diff
@@ -908,8 +909,11 @@ groups:
     settings:
       ReconcileMembers: "true"
     members:
+      - cecilerobertm@gmail.com
       - davanum@gmail.com
+      - david@devigned.com
       - nicklaneovi@gmail.com
```
[the remark this answers is not in the document]
Comment (randomvariable) [reply groups/groups.yaml]: Done

**Pull request merged**